UniversalDashboard.Forge.psm1

```diff
@@ -156,14 +164,14 @@ function New-UDDesktopApp {
 function Copy-UniversalDashboard {
     param($OutputPath)
 
-    $UniversaDashboard = Get-Module -Name UniversalDashboard -ListAvailable
+    $UniversalDashboard = (Get-Module -Name UniversalDashboard -ListAvailable)[0]
```

Comment:
This will get just the latest version so previously installed versions don't get copied as well.